server/schedule/filter/filters.go

```diff
@@ -358,6 +358,8 @@ type StoreStateFilter struct {
 	TransferLeader bool
 	// Set true if the schedule involves any move region operation.
 	MoveRegion bool
+	// Set true if allows temporary states.
+	AllowTemporaryStates bool
```

Comment:
When will it be true?

Reply:
next version of `replicaChecker` will use it.

Reply:
got it